layouts/partials/summary.html

```diff
@@ -0,0 +1,12 @@
+<div class="pageinfo pageinfo-primary">
```

Comment:
`tag-summary.html` might be a better name, given it has an explicit purpose

Reply:
i also note that most files in docsy use dash as a separator rather than underscore, though i don't know if that was intended to be a rule; would be nice to be consistent